File: Filters/src/entity/Customer.js
function Customer(customerID,customerName,customerAddress,customerTeleNumber){
    var __cusId=customerID;
    var __cusName=customerName;
    var __cusAddress=customerAddress;
    var __cusTeleNumber=customerTeleNumber;

    this.setCustomerID=function (id){
        __cusId=id;
    }

    this.getCustomerID=function (){
        return __cusId;
    }
    this.setCustomerName = function (name) {
        __cusName = name;
    };

    this.getCustomerName = function () {
        return __cusName;
    };

    this.setCustomerAddress = function (address) {
        __cusAddress = address;
    };

    this.getCustomerAddress = function () {
        return __cusAddress;
    };

    this.setCustomerTeleNumber = function (teleNumber) {
        __cusTeleNumber = teleNumber;
    };

    this.getCustomerTeleNumber = function () {
        return __cusTeleNumber;
    };

}

File: BackEnd/src/entity/Customer.java
package entity;

public class Customer {
    private String customerID;
    private String customerName;
    private String customerAddress;
    private String customerTeleNumber;

    public Customer(String customerID, String customerName, String customerAddress, String customerTeleNumber) {
        this.customerID = customerID;
        this.customerName = customerName;
        this.customerAddress = customerAddress;
        this.customerTeleNumber = customerTeleNumber;
    }

    public Customer() {

    }

    public String getCustomerID() {
        return customerID;
    }

    public void setCustomerID(String customerID) {
        this.customerID = customerID;
    }

    public String getCustomerName() {
        return customerName;
    }

    public void setCustomerName(String customerName) {
        this.customerName = customerName;
    }

    public String getCustomerAddress() {
        return customerAddress;
    }

    public void setCustomerAddress(String customerAddress) {
        this.customerAddress = customerAddress;
    }

    public String getCustomerTeleNumber() {
        return customerTeleNumber;
    }

    public void setCustomerTeleNumber(String customerTeleNumber) {
        this.customerTeleNumber = customerTeleNumber;
    }

}
